### PR TITLE
small style clean up for hard coded floats.

### PR DESCRIPTION
**Pull request by Br3nda**
small style clean up for hard coded floats.

### CODE REVIEW

File: mobile/src/main/java/org/openhab/habdroid/model/OpenHABItem.java [hunk 1]
```diff
@@ -148,11 +148,11 @@ public Float getStateAsFloat() {
 		Float result;
 		// For uninitialized/null state return zero
 		if (state == null) {
-			result = new Float(0);
+			result = 0f;
```
Comment (watou) [inline mobile/src/main/java/org/openhab/habdroid/model/OpenHABItem.java]: This also has the benefit of resulting in `Float.valueOf(0f)` being generated by the compiler when autoboxing the native `float`, where the JavaDoc says "[valueOf] should generally be used in preference to the constructor Float(float), as this method is likely to yield significantly better space and time performance by caching frequently requested values.".

Comment (Br3nda) [inline mobile/src/main/java/org/openhab/habdroid/model/OpenHABItem.java]: also, the android-studio linter will stop glaring yellow at those lines.

**Comment by digitaldan** [comment]
Thanks again for the PR!

File: mobile/src/main/java/org/openhab/habdroid/model/OpenHABItem.java (hunk 1 above)
Comment (digitaldan) [inline mobile/src/main/java/org/openhab/habdroid/model/OpenHABItem.java]: Should line 164 also receive this change?

**Comment by digitaldan** [comment]
Hi I have not forgotten about your PR's !  I'm currently trying to get our cloudbee builds back up, as soon as that happens I will merge, thanks!

**Pull request merged**